includes/JumpstartSitesEngineering.php

```diff
@@ -664,5 +672,82 @@ private function fetch_jse_content_beans($endpoint) {
     $importer->import_content_beans();
 
   }
+  
+  /**
+   * Installs and configures the default users for jumpstart
+   * @param  [array] $install_state [the current installation state]
+   */
+  public function jse_install_jumpstart_users(&$install_state) {
+
+    drush_log('JSE - Starting install users', 'status');
+
+    // Need this for UI install.
+    require_once DRUPAL_ROOT . '/includes/password.inc';
+    $install_vars = variable_get('stanford_jumpstart_install', array());
+    $config_form_data = $install_state['forms']['install_configure_form'];
+
+    // Get some stored variables.
+    if ($install_state['interactive']) {
+      $full_name  = isset($install_vars['full_name']) ? $install_vars['full_name'] : "School of Engineering";
+      $sunetid    = isset($install_vars['sunetid']) ? $install_vars['sunetid'] : 'jse-admins';
+    }
+    else if (function_exists('drush_get_option')) {
+      $full_name  = isset($config_form_data['stanford_sites_requester_name']) ? $config_form_data['stanford_sites_requester_name'] : drush_get_option('full_name', 'Engineering');
+      $sunetid    = isset($config_form_data['stanford_sites_requester_sunetid']) ? $config_form_data['stanford_sites_requester_sunetid'] : drush_get_option('sunetid', 'jse-admins');
+    }
+    else {
+      $full_name  = "Engineering";
+      $sunetid    = "jse-admins";
+    }
+
+    // add WMD user (site owner)
+    // drush waau $sunetid --name="$fullname"
+    $sunet = strtolower(trim($sunetid));
+    $authname = $sunet . '@stanford.edu';
+
+    $sunet_role = user_role_load_by_name('SUNet User');
+    $owner_role = user_role_load_by_name('site owner');
+    $editor_role = user_role_load_by_name('editor');
+    $admin_role = user_role_load_by_name('administrator');
+    $member_role = user_role_load_by_name('site member');
+
+    // Change user 1, currently admin, to jse-admins
```

Comment:
You're not changing the user name here, only the email address. That may be what you want to do, I'm not sure.

```
% drush uinf 1


 User ID       :  1                             
 User name     :  admin                         
 User mail     :  jse-admins@lists.stanford.edu 
 User roles    :  authenticated user            
                  SUNet User                    
                  administrator                 
 User status   :  active           
```